### ISSUE TITLE
Deployment script: use git pull instead of extracting from zip

### ISSUE THREAD

**Issue by jvanloov** [0x20/spb#9]
'git pull' doesn't overwrite local changes, so it should be OK to use for deployment as well; it should leave the config files alone.

**Comment by biertie** [comment]
git pull doesn't do that since the latest patch by Sasja I think. So we can already close this one, am I right?

**Comment by jvanloov** [comment]
No, the idea is to modify the deployment script (devops/deploy_to_{test,prod}.sh)

Currently, running this script will retrieve a ZIP archive from github, from which various .py, .html,… files are extracted and copied into the directory of the {test,prod} installation on 172.22.32.{6,4}. I did it like that because I wanted to be specific about not overwriting configuration files specific for test or prod. 

However, I noticed that if you do ‘git pull’ to update your local code, local modifications are left alone. So the scripts can be simplified by simply running ‘git pull’ on the test or prod installations. (Of course it leaves local modifications alone, it is meant to only pull in remote changes, I should have realized that, but hey…)

Please leave the issue open until the scripts are updated...

J.

Joris Van Looveren
joris@van-looveren.net mailto:joris@van-looveren.net
+32 496 45 67 01
